exampleSite/content/posts/tests/mermaid-tests/index.en.md

```diff
@@ -44,8 +41,7 @@ Future task               :         des3, after des2, 5d
 Future task2               :         des4, after des3, 5d
 {{< /mermaid >}}
 
-{{< mermaid >}}
-classDiagram
+{{< mermaid >}}classDiagram
```

Comment:
This issue is only related to `classDiagram`. Adding `\r\n` before classDiagram will cause rendering issue. So for consistency, all `\r\n` are removed in markdown.

Again, I think this is a bug by mermaid itself. But anyway.